packages/cli/src/lib/commands/schema-util.ts

```diff
@@ -73,7 +79,29 @@ export const validateFinal = (schemaAppRequest: InputData): true | string => {
 export const appLinksDefSummarize = (value?: ViperAppLinks): string =>
 	clipToMaximum(`android: ${value?.android}, ios: ${value?.ios}`, maxItemValueLength)
 
-export const buildInputDefinition = (command: SmartThingsCommandInterface, initialValue?: SchemaAppRequest): InputDefinition<InputData> => {
+export const organizationDef = (organizations: OrganizationResponse[]): InputDefinition<string | undefined> => {
+	if (organizations.length < 1) {
+		return undefinedDef
+	}
+	if (organizations.length === 1) {
+		return staticDef(organizations[0].organizationId)
+	}
+
+	const choices = organizations
+		.map(organization => ({
+			name: organization.name,
+			value: organization.organizationId,
+		}))
+
+	const helpText = 'The organization to which the Schema connector should be associated.'
```

Comment:
The smallest of nits:
```suggestion
	const helpText = 'The organization with which the Schema connector should be associated.'
```
I am not 100% confident on this one, but I'm wondering if this should be "with" instead of "to" - @Sitlintac what do you think?

Reply:
I considered that question too and left it as-is because I couldn't decide if this was one of those instances where "associated to" is correct.  I think it might be (if you consider an organization to be of higher rank than a connector, which is plausible to me because each connector belongs to one organization but an organization can have multiple connectors), but I agree that "with" is almost always the better choice.

Reply:
I like "with" better. I think I only had "to" in there because I had started out using the verb "connected" instead of "associated". Fixed.